sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ParDo.java

```diff
@@ -932,10 +928,7 @@ public void populateDisplayData(Builder builder) {
 
     @Override
     public PCollectionTuple expand(PCollection<? extends InputT> input) {
-      checkArgument(
-          !isSplittable(getFn()),
-          "%s does not support Splittable DoFn",
-          input.getPipeline().getOptions().getRunner().getName());
+      // SplittableDoFn should be forbidden on the runner-side.
```

Comment:
Are there JIRAs for the other runners to reject it? Otherwise I think this will just be silently busted.